index.md

```diff
@@ -7,86 +7,54 @@ title: MDAnalysis
-
-
-## Supporting ##
-
-If you like MDAnalysis and want to support our mission, please
-consider making a donation to support our efforts.
+ The **MDAnalysis Project** develops open-source tools for the analysis of molecular simulation data, providing researchers with efficient and accessible solutions for studying molecular structures and dynamics. The project is a fiscally sponsored initiative of [NumFOCUS][], a nonprofit organization that supports open-source scientific computing.
 
-{{ site.numfocus.donate_button }}
+At the core of the project is MDAnalysis, **an open-source Python library** for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{ site.docs.userguide.url }}/formats/index.html#formats). MDAnalysis enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]).
+
+With MDAnalysis, you can access atomic coordinates as [NumPy][] arrays, providing a flexible and efficient framework for complex analysis tasks. The library includes powerful [atom selection commands]({{ site.docs.userguide.url }}/selections.html) for extracting subsets of structures and supports trajectory transformations (e.g., fitting to a reference structure).
+
+Learn more about our **mission, development, team, and governance** on the [About MDAnalysis]({{ site.baseurl }}/about/) page.
+
+## Get Started  
+
+If you're new to MDAnalysis, explore the following resources to install the library, access tutorials, and dive into the documentation:
+
+- [Getting Started]({{ site.baseurl }}/pages/getting_started/)  
+- [Learning MDAnalysis]({{ site.baseurl }}/pages/learning_MDAnalysis)  
+- [Documentation]({{ site.baseurl }}/pages/documentation)  
+
+## Community 
+
+MDAnalysis is driven by an active **community of users and contributors**. Stay updated and get involved through the following pages:  
+
+<!-- TODO: Add link [Get Involved]({{ site.baseurl }}/pages/about/#get-involved)
+ -->
+- [Community]({{ site.baseurl }}/pages/community/) &mdash; Get involved, ask questions, and collaborate.
+- [Blog]({{ site.baseurl }}/blog) &mdash; Stay updated with development news and community highlights.
+- [Events]({{ site.baseurl }}/pages/events/) &mdash; Join workshops, conferences, and mentoring programs.
+
+## Contributing 
 
-(Donations are made through [our fiscal sponsor][], [NumFOCUS][], which is
-a 501(c)(3) non-profit charity in the United States; as such,
-donations to NumFOCUS are tax-deductible as allowed by law.  As with
-any donation, you should consult with your personal tax adviser or the
-IRS about your particular tax situation.)
```

Comment:
Should remain next to donate button.

Reply:
YOu can enclose in `<small>` tags or a CSS to decrease font size, or make it a footnote, but this information should be somewhere near the donate button.